docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -293,8 +293,8 @@ Update of the software will be managed by the swupd tool::
 Latest Releases via Pip
 +++++++++++++++++++++++
 
-Ansible can be installed via "pip", the Python package manager.  If 'pip' isn't already available in
-your version of Python, you can get pip by::
+Ansible can be installed via ``pip``, the Python package manager.  If ``pip`` isn't already available in
+your version of Python, you can get ``pip`` by::
 
    $ sudo easy_install pip
```

Comment:
Since you're updating docs you should probably update the pip install instructions:

https://pip.pypa.io/en/stable/installing/#installing-with-get-pip-py